code/src/styling/Button.js

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
```

Comment:
Props for using styled-components! And for creating a reusable button component, this makes your code cleaner, nice job!🌈